Plugins/PsxReverb/PsxReverb.cpp

```diff
@@ -274,6 +288,32 @@ void PsxReverb::DoEditorSetup() noexcept {
     };
 }
 
+//------------------------------------------------------------------------------------------------------------------------------------------
+// Called when a parameter changes
+//------------------------------------------------------------------------------------------------------------------------------------------
+void PsxReverb::OnParamChangeUI(int paramIdx, EParamSource source) noexcept {
```

Comment:
This callback was also added to PR https://github.com/BodbDearg/PlayStation1Vsts/pull/1 so it will probably conflict during merge.